chirpstack/src/api/oauth2.rs

```diff
@@ -129,9 +135,11 @@ pub async fn get_user(code: &str, state: &str) -> Result<User> {
     let conf = config::get();
     let provider = conf.user_authentication.oauth2.provider.clone();
     let userinfo_url = conf.user_authentication.oauth2.userinfo_url.clone();
+    let assume_email_verified = conf.user_authentication.oauth2.assume_email_verified.clone();
```

Comment:
There is no need to use `.clone()` for `bool` types as they implement the `Copy` trait (this is not the case for `String` types).